bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -21,7 +21,7 @@
 {% endblock %}
 
 {% block site_header %}
-  {% with hide_nav_download_button=True %}
+  {% with hide_nav_cta=False, hide_nav_download_button=False, hide_nav_get_account_button=True %}
```

Comment:
Both `hide_nav_cta=False` and  `hide_nav_download_button=False` are superfluous here.

This should be just `hide_nav_get_account_button=True`.